libr/core/core.c

```diff
@@ -1029,6 +1034,97 @@ static const char *radare_argv[] = {
 	NULL
 };
 
+static void autocomplete_ms_path(RLineCompletion *completion, RCore *core, const char *str, const char *path) {
+	char *lpath = NULL, *dirname = NULL , *basename = NULL;
+	char *home = NULL, *p = NULL;
+  char *pwd = *(core->rfs->cwd);
+	int n = 0;
+	RList *list;
+	RListIter *iter;
+  RFSFile *file;
+
+	if (!path) {
+		goto out;
+	}
+
+	lpath = r_str_new (path);
```

Comment:
You might want to split this logic into the separate function

Reply:
I separated it into two methods getting mount points and files